apache-airflow-mixin/README.md

```diff
@@ -0,0 +1,124 @@
+# Apache Airflow Mixin
+
+The Apache Airflow mixin is a set of configurable Grafana dashboards and alerts.
+
+The Apache Airflow mixin contains the following dashboards:
+
+- Apache Airflow overview
+
+and the following alerts:
+
+- ApacheAirflowStarvingPoolTasks
+- ApacheAirflowDAGScheduleDelayWarningLevel
+- ApacheAirflowDAGScheduleDelayCriticalLevel
+- ApacheAirflowDAGFailures
+
+## Apache Airflow Overview
+
+The Apache Airflow overview dashboard provides details on DAG failures, DAG durations, task failures, task durations, scheduler details, executor tasks, and pool task slots for an Apache Airflow system.
+
+![Apache Airflow overview dashboard (1/2)](TBD)
+![Apache Airflow overview dashboard (2/2)](TBD)
+
+Apache Airflow system logs are enabled by default in the `config.libsonnet` and can be removed by setting `enableLokiLogs` to `false`. Then run `make` again to regenerate the dashboard:
+
+```
+{
+  _config+:: {
+    enableLokiLogs: false,
+  },
+}
+```
+
+In order for the selectors to properly work for system logs ingested into your logs datasource, please also include the matching `instance` and `job` labels onto the [scrape_configs](https://grafana.com/docs/loki/latest/clients/promtail/configuration/#scrape_configs) as to match the labels for ingested metrics. It is also important to add matching `dag_id` and `task_id` for the task logs to match the labels for ingested metrics as well as add labels for `dag_file` for the scheduler logs to allow for filtering in the dashboard.
+
+```yaml
+scrape_configs:
+- job_name: system
+  static_configs:
```

Comment:
just wondering, should the `job_name` here match the technology like `apache-airflow`

Reply:
@armstrmi good catch. i'll make that change

Reply:
Should be fixed